user-guide/MACS.rst

```diff
@@ -1,26 +1,41 @@
-Phase 1
--------
+Multi-Architecture Comparison System
+------------------------------------
 
-The first phase of the Isambard system hosts many nodes of different architectures:
+Isambard MACS hosts many nodes of different architectures:
 
-* 2x Login nodes with Intel Xeon CPU
-* 4x GPU nodes with 2x Nvidia P100 "Pascal" GPUs and Intel Xeon E5-2695 v4 "Broadwell" CPU
-* 8x Intel Xeon Phi with "Knights Landing" 7210 CPU
-* 2x IBM Power 9 nodes
-* * Including 2x Nvidia V100 "Volta" GPU with NVLink
+* 2x Login nodes with Intel Xeon "Broadwell" CPU
+* 4x Nvidia Pascal GPU nodes with 2x Nvidia P100 "Pascal" GPUs and Intel Xeon E5-2695 v4 "Broadwell" CPU
+* 4x Nvidia Volta GPU nodes with 1x Nvidia V100 "Volta" GPU and Intel Xeon Gold 6230 "Cascade Lake (CXL)" CPU
+* 4x AMD Epyc 7742 "Rome" CPUs
+* 4x Intel Xeon 6230 "Cascade Lake (CXL)" CPU
+* 8x Intel Xeon Phi "Knights Landing" 7210 CPU
+* 2x IBM Power 9 nodes each with 2x Nvidia V100 "Volta" GPU
 
-Login, Pascal & KNL nodes run Red Hat Enterprise Linux 7.6 with Cray software stacks.
+Most MACS nodes run Red Hat Enterprise Linux 7.x with the Cray software stack.
 
-Power 9 nodes run Red Hat Enterprise Linux for Little-Endian 7.5 with IBM C/C++/Fortran compilers & IBM Power AI.
+Power 9 nodes run Red Hat Enterprise Linux (Little-Endian) with IBM compilers & IBM Power AI.
 
 All nodes are connected via 100 Gigabit EDR Infiniband. The login nodes are connected to the Internet via a 10 Gigabit link to the `Janet Network <https://www.jisc.ac.uk/janet>`_.
 
-Intel KNL
-=========
+Nvidia GPU
+==========
 
-`Intel Xeon Phi "Knights Landing" 7210 CPU <https://ark.intel.com/products/94033/Intel-Xeon-Phi-Processor-7210-16GB-1_30-GHz-64-core>`_ @ 1.30GHz (1.50GHz Turbo) with AVX-512.
+Isambard hosts two generations of Nvidia GPUs: `P100 "Pascal" <https://www.nvidia.com/en-us/data-center/pascal-gpu-architecture/>`_ & `V100 "Volta" <https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/>`_.
+
+AMD Epyc
+========
+
+`AMD Epyc 7742 "Rome" <https://www.amd.com/en/products/cpu/amd-epyc-7742>`_ 2.25GHz 64-core CPU nodes with DDR4-3200Mhz over 8 memory channels.
```

Comment:
Should be mention threading and multi-socket nature of the nodes?

Reply:
I'll leave that to someone else 😉